src/frontend/src/components/AuthorResult/AuthorResult.jsx

```diff
@@ -81,30 +78,23 @@ export default function AuthorResult() {
                   <ListItemIcon className={classes.icons}>
                     <PermContactCalendarIcon />
                   </ListItemIcon>
-                  <ListItemText className={classes.font} primary="Username" />
+                  <ListItemText className={classes.font} primary={`Username: ${author}`} />
                 </ListItem>
                 <ListItem button className={classes.infoLine}>
                   <ListItemIcon className={classes.icons}>
                     <EventIcon />
                   </ListItemIcon>
-                  <ListItemText className={classes.font} primary="Date of Last Post: " />
+                  <ListItemText
+                    className={classes.font}
+                    primary={`Date of Last Post: ${postDate}`}
+                  />
                 </ListItem>
-                <ListItem button className={classes.infoLine} onClick={handleClick}>
+                <ListItem button className={classes.infoLine} onClick={handleLatestPostClick}>
```

Comment:
If the user is supposed to click on it, and it opens content, that's an `<a>`.  Why use a `<button>`?